disnake/member.py

```diff
@@ -293,6 +293,7 @@ class Member(disnake.abc.Messageable, _UserTag):
         banner: Optional[Asset]
         accent_color: Optional[Colour]
         accent_colour: Optional[Colour]
+        avatar_decoration: Optional[Asset]
```

Comment:
guild-specific avatar decorations were added recently: https://github.com/discord/discord-api-docs/pull/6464/commits/16cbede18fd8b5805d74dfd935208501074d738a
would you mind implementing those in this PR as well? Otherwise I can create a separate one later.

Reply:
fc463a87e24c0251edd0cb8bb5ea016efd65e0fc